src/App.jsx

```diff
@@ -1,33 +1,64 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getPrepearedMovies(movies, query) {
+  let prepearedMovies = [...movies];
```

Comment:
```suggestion
  let preparedMovies = [...movies];
```
Typo in "prepared". Fix all cases.
Also, filter is a non-mutating method, so there is actually no need to create a copy